app/src/main/java/com/android/voyageur/ui/overview/AddTrip.kt

```diff
@@ -230,10 +297,44 @@ fun AddTripScreen(
                   createTripWithImage("")
                 }
               },
-              enabled = name.isNotBlank() && startDate.isNotBlank() && endDate.isNotBlank(),
+              enabled =
+                  name.isNotBlank() &&
+                      formattedStartDate.isNotBlank() &&
+                      formattedEndDate.isNotBlank(),
               modifier = Modifier.fillMaxWidth().padding(16.dp).testTag("tripSave")) {
                 Text("Save Trip")
               }
         }
       }
 }
+
+enum class DateField {
+  START,
+  END
+}
+
+@OptIn(ExperimentalMaterial3Api::class)
```

Comment:
This should be moved to a separate package in the future as we migh reuse it for some other screens.

Reply:
It's not urgent for now, but we need it again, please move it in a shared file.